subsystem/kubeapi_test.go

```diff
@@ -294,6 +294,13 @@ func getAgentCRD(ctx context.Context, client k8sclient.Client, key types.Namespa
 	return agent
 }
 
+func getAgentClusterImageSetCRD(ctx context.Context, client k8sclient.Client, key types.NamespacedName) *hivev1.ClusterImageSet {
```

Comment:
Why "Agent" ClusterImageSet is not an Agent resource

Reply:
fixed